tests/cicd/main_build.yml

```diff
@@ -267,162 +271,13 @@ jobs:
-      # find the latest model which we just trained
-      # if we're running on a build VM
-      model_dir=$(ls -td output/unet/section_depth/* | head -1)
-      # if we're running in a checked out git repo
-      [[ -z ${model_dir} ]] && model_dir=$(ls -td output/$(git rev-parse --abbrev-ref HEAD)/*/unet/section_depth* | head -1)
-      model=$(ls -t ${model_dir}/*.pth | head -1)
-      # try running the test script
-      { python test.py 'DATASET.ROOT' '/home/alfred/data_dynamic/dutch_f3/data' \
-                       'TEST.SPLIT' 'Both' 'TRAIN.MODEL_DIR' 'section_depth' \
-                       'TEST.MODEL_PATH' ${model} \
-                       'WORKERS' 1 \
-                       --cfg=configs/unet.yaml --debug ; echo "$?" > "$dir/$BASHPID"; }
-      pids+=" $!"
-      # export CUDA_VISIBLE_DEVICES=2
-      # find the latest model which we just trained
-      # if we're running on a build VM
-      model_dir=$(ls -td output/seresnet_unet/section_depth/* | head -1)
-      # if we're running in a checked out git repo
-      [[ -z ${model_dir} ]] && model_dir=$(ls -td output/$(git rev-parse --abbrev-ref HEAD)/*/seresnet_unet/section_depth/* | head -1)
-      model=$(ls -t ${model_dir}/*.pth | head -1)
-      # try running the test script
-      { python test.py 'DATASET.ROOT' '/home/alfred/data_dynamic/dutch_f3/data' \
-                       'TEST.SPLIT' 'Both' 'TRAIN.MODEL_DIR' 'section_depth' \
-                       'TEST.MODEL_PATH' ${model} \
-                       'WORKERS' 1 \
-                       --cfg=configs/seresnet_unet.yaml --debug ; echo "$?" > "$dir/$BASHPID"; }
-      pids+=" $!"
-      # export CUDA_VISIBLE_DEVICES=3
-      # find the latest model which we just trained
-      # if we're running on a build VM
-      model_dir=$(ls -td output/hrnet/section_depth/* | head -1)
-      # if we're running in a checked out git repo
-      [[ -z ${model_dir} ]] && model_dir=$(ls -td output/$(git rev-parse --abbrev-ref HEAD)/*/hrnet/section_depth/* | head -1)
-      model=$(ls -t ${model_dir}/*.pth | head -1)
-      # try running the test script
-      { python test.py 'DATASET.ROOT' '/home/alfred/data_dynamic/dutch_f3/data' \
-                       'TEST.SPLIT' 'Both' 'TRAIN.MODEL_DIR' 'section_depth' \
-                       'MODEL.PRETRAINED' '/home/alfred/models/hrnetv2_w48_imagenet_pretrained.pth' \
-                       'TEST.MODEL_PATH' ${model} \
-                       'WORKERS' 1 \
-                       --cfg=configs/hrnet.yaml --debug ; echo "$?" > "$dir/$BASHPID"; }
-      pids+=" $!"
-
-      # wait for completion
-      wait $pids || exit 1
-
-      # check if any of the models had an error during execution
-      # Get return information for each pid
-      for file in "$dir"/*; do
-        printf 'PID %d returned %d\n' "${file##*/}" "$(<"$file")"
-        [[ "$(<"$file")" -ne "0" ]] && exit 1 || echo "pass"
-      done
-
-      # Remove the temporary directory
-      rm -r "$dir"
-
-      echo "PASSED"
```

Comment:
Sorry I might be missing something obvious, but why are thes tests removed?

Reply:
@yalaudah I mark this as resolved per our chat :)